pootle/apps/pootle_store/views.py

```diff
@@ -131,71 +131,20 @@ def get_step_query(request, units_queryset):
                 pass
 
         if unit_filter:
-            match_queryset = units_queryset.none()
-
-            if unit_filter == 'all':
-                match_queryset = units_queryset
-            elif unit_filter == 'translated':
-                match_queryset = units_queryset.filter(state=TRANSLATED)
-            elif unit_filter == 'untranslated':
-                match_queryset = units_queryset.filter(state=UNTRANSLATED)
-            elif unit_filter == 'fuzzy':
-                match_queryset = units_queryset.filter(state=FUZZY)
-            elif unit_filter == 'incomplete':
-                match_queryset = units_queryset.filter(
-                    Q(state=UNTRANSLATED) | Q(state=FUZZY),
-                )
-            elif unit_filter == 'suggestions':
-                match_queryset = units_queryset.filter(
-                    suggestion__state=SuggestionStates.PENDING).distinct()
-            elif unit_filter in ('my-suggestions', 'user-suggestions'):
-                match_queryset = units_queryset.filter(
-                    suggestion__state=SuggestionStates.PENDING,
-                    suggestion__user=user,
-                ).distinct()
-                sort_on = 'suggestions'
```

Comment:
With the new code this is no longer set. Same for `'my-submissions', 'user-submissions'` below.